### ISSUE TITLE
Support for private ips

### ISSUE THREAD

**Issue by kaspergrubbe** [linode/packer-plugin-linode#75]
#### Description

I'm using private ips for the Linodes that I launch with Terraform, and it would be nice to be able to configure that for the Linode that Packer sets up.

#### Use Case(s)

I am building Consul servers with Packer, and I am using the private IPs for communication, this works fine for Linodes that have `private_ip = true` (through Terraform), but Consul throws errors if there is no private IP attached.

During my Packer build of my Consul images I run a sanity check: `/usr/local/bin/consul validate /etc/consul.d/consul.json` however this fails with this error:

```bash
==> linode: Config validation failed: No private IPv4 address found
```

If I could configure the linode to have private networking enabled I could catch configuration mistakes early.

**Comment by lgarber-akamai** [comment]
@kaspergrubbe Thanks for the feature request! 

I've added a ticket to our internal backlog to add support for private IPv4 address allocation and will let you know when we have any updates to share 🙂

**Comment by lgarber-akamai** [comment]
Hello @kaspergrubbe!
This functionality has been released in `v1.1.0`.
